cli/src/cli_util.rs

```diff
@@ -1290,6 +1290,11 @@ impl WorkspaceCommandHelper {
             .expect("parse error should be confined by WorkspaceCommandHelper::new()")
     }
 
+    pub fn format_short_change_id(&self) -> TemplateRenderer<'_, Commit> {
+        self.parse_commit_template("format_short_change_id(self.change_id())")
+            .expect("syntax is valid")
```

Comment:
`format_short_change_id()` can be overridden by user, and may have syntax error.

Reply:
Ah okay, I thought it only parsed this snippet specifically (anything else being caught upfront)